tests/python_test.py

```diff
@@ -600,8 +617,10 @@ def test_init_lockspace_parse_args(no_sanlock_daemon, name):
 
 
 @pytest.mark.parametrize("name", LOCKSPACE_OR_RESOURCE_NAMES)
-def test_init_resource_parse_args(no_sanlock_daemon, name):
-    disks = [("path", 0)]
+@pytest.mark.parametrize("filename,encoding", FILE_NAMES_NO_XFAILS)
+def test_init_resource_parse_args(no_sanlock_daemon, name, filename, encoding):
+    path = util.generate_path("/tmp/", filename, encoding)
+    disks = [(path, 0)]
     with raises_sanlock_errno(errno.ENOENT):
```

Comment:
The tests changes should be separate patch. You are missing here improvement of the tests and changing the code. This is very likely to fail creating tests that do not work. I will have more confidence when I see the tests fail - before this change.

Reply:
this is what we talked about before this PR - to change each test as part
of each relevant patch

Reply:
unless you meant the case where the args parse test is added the first time.

Reply:
We discussed that the patch fixing a failing test should enable the failing test, but here you modified the test and this make the code harder to review. If you think we need to modify the tests by adding parameters for path, why not submit this change as a separate commit before you modify the code?

Reply:
The only test change that should be included in this commit is replacing:

    @pytest.mark.parametrize("filename,encoding", FILE_NAMES)

With:

    @pytest.mark.parametrize("filename,encoding", FILE_NAMES_NO_XFAILS)